src/python/pants/backend/native/tasks/link_shared_libraries.py

```diff
@@ -31,18 +31,26 @@ class LinkSharedLibraryRequest(datatype([
     'output_dir',
     ('external_lib_dirs', tuple),
     ('external_lib_names', tuple),
+    ('external_static_archive_paths', tuple),
 ])):
 
   @classmethod
   def with_external_libs_product(cls, external_libs_product=None, *args, **kwargs):
     if external_libs_product is None:
       lib_dirs = ()
       lib_names = ()
+      static_archive_paths = ()
     else:
       lib_dirs = (external_libs_product.lib_dir,)
       lib_names = external_libs_product.lib_names
+      static_archive_paths = external_libs_product.static_archive_paths
 
-    return cls(*args, external_lib_dirs=lib_dirs, external_lib_names=lib_names, **kwargs)
+    return cls(
```

Comment:
This will conflict with the change I was working on. Are you ok with waiting until that is in, and then rebasing?

Reply:
Yes!